components/common/ConnectWallet/WalletAssetMenu.vue

```diff
@@ -61,27 +61,29 @@ import { NeoDropdown, NeoDropdownItem, NeoIcon } from '@kodadot1/brick'
 import { langsFlags, setUserLocale } from '@/utils/config/i18n'
 
 const { urlPrefix } = usePrefix()
-// const { isAssetHub } = useIsChain(urlPrefix)
+const { isBase } = useIsChain(urlPrefix)
 const { neoModal } = useProgrammatic()
 
-const menus = ref([
-  {
-    label: 'Transfer',
-    to: `/${urlPrefix.value}/transfer`,
-  },
-  {
-    label: 'Teleport Bridge',
-    to: `/${urlPrefix.value}/teleport`,
-  },
-  {
-    label: 'Onchain Identity',
-    to: '/identity',
-  },
-  {
-    label: 'Migrate',
-    to: '/migrate',
-  },
-])
+const menus = computed(() =>
+  [
+    {
+      label: 'Transfer',
+      to: `/${urlPrefix.value}/transfer`,
+    },
+    {
+      label: 'Teleport Bridge',
+      to: `/${urlPrefix.value}/teleport`,
+    },
+    {
+      label: 'Onchain Identity',
+      to: '/identity',
+    },
+    {
+      label: 'Migrate',
+      to: '/migrate',
+    },
+  ].filter((menu) => !isBase.value || menu.label !== 'Transfer'),
+)
```

Comment:
None of this is available on base :) also think it was solved by @hassnian

Reply:
https://github.com/kodadot/nft-gallery/blob/578f43b1ae37c9338e6eaf9daea90684de40b44d/components/common/ConnectWallet/WalletAssetMenu.vue#L70

Reply:
Then I guess we could close this issue as it has been implemented on other PR.